test/Makefile

```diff
@@ -31,7 +31,7 @@ gcp-hack: ../artifacts/test.img.tar.gz
 ../artifacts/test.img.tar.gz:
 	rm -rf ../artifacts
 	mkdir -p ../artifacts
-	$(MOBY) build --pull -name ../artifacts/test hack/test.yml
+	$(MOBY) build -output gcp-img --pull -name ../artifacts/test hack/test.yml
```

Comment:
this mixes `-` and `--` for command line arguments. Well, it did so before as well, but might as well not do that